src/Stripe.Tests.XUnit/transfer_reversals/_fixture.cs

```diff
@@ -54,7 +54,7 @@ public transfer_reversals_fixture()
             TransferReversal = service.Create(Transfer.Id, TransferReversalCreateOptions);
             TransferReversalUpdated = service.Update(Transfer.Id, TransferReversal.Id, TransferReversalUpdateOptions);
             TransferReversalRetrieved = service.Get(Transfer.Id, TransferReversal.Id);
-            TransferReversalList = service.List(Transfer.Id, new StripeListOptions());
+            TransferReversalList = service.List(Transfer.Id, TransferReversalListOptions);
```

Comment:
why don't you need a new here?

Reply:
Oops! I didn't see that `TransferReversalListOptions` was declared but never initialized. This is now fixed.